### PR TITLE
Update python drivers (options) docs with code examples

### PR DESCRIPTION
**Pull request by pmartinez1**
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description
This PR adds/moves code examples for python under the Drivers - options section of the site.

### Motivation and Context

The examples should be helpful for those who are wanting to use Selenium.

### Types of changes

- [x ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Documentation, Enhancement

___

### **Description**
- Added multiple test functions for various ChromeOptions settings in `examples/python/tests/drivers/test_options.py`.
- Updated Python code examples across multiple documentation files to use `gh-codeblock` with specific line references.
- Replaced inline Python code with references to the test file in the options documentation.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests</strong></td><td><details><summary>1 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>test_options.py</strong><dd><code>Add comprehensive test cases for ChromeOptions settings</code>&nbsp; &nbsp; </dd></summary>
<hr>

examples/python/tests/drivers/test_options.py

<li>Added multiple test functions for various ChromeOptions settings.<br> <li> Included tests for capabilities, timeouts, unhandled prompts, window <br>rect, strict file interactability, and proxy settings.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-2634464e892392fa3ffdf1940368c555053d49f5d1491886253b739676ea7730">+68/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr><tr><td><strong>Documentation</strong></td><td><details><summary>8 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>_index.en.md</strong><dd><code>Update Python code examples with specific line references</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/_index.en.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-10ff84d69cea7383c29ebfd0e4934cda8a3d6111bfc937897b1f446c4f4781b5">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>_index.ja.md</strong><dd><code>Update Python code examples with specific line references (Japanese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/_index.ja.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-bfbaccc3ea80f5b819613809c7324862ee335c163d6e474b07a0d3aa7e48740e">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>_index.pt-br.md</strong><dd><code>Update Python code examples with specific line references (Portuguese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/_index.pt-br.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-86b91e7aed382ec5c668cf7a26e77cc125d1b983696d9fa5962ce8c47c598fc9">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>_index.zh-cn.md</strong><dd><code>Update Python code examples with specific line references (Chinese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/_index.zh-cn.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-271afe3b24802a1c6427f915a6ded1b3f370a63eda9ee97b38820971149f4563">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.en.md</strong><dd><code>Update Python options documentation with specific line references</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.en.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br> <li> Replaced inline Python code with references to the test file.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-62bab9f28d27a7f462881a28eca384a0e534a730f849248f396d171820bc748e">+14/-28</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.ja.md</strong><dd><code>Update Python options documentation with specific line references </code><br><code>(Japanese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.ja.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br> <li> Replaced inline Python code with references to the test file.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-2ed475220bde1368cfa3883d71db179409ddaf9ccbaea847ed3f6ffe816321e4">+14/-28</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.pt-br.md</strong><dd><code>Update Python options documentation with specific line references </code><br><code>(Portuguese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.pt-br.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br> <li> Replaced inline Python code with references to the test file.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-6eaa598a13e9d64c6afbd54b84aef470520557f08c4d183ffb28ceb9ed04d0bb">+14/-28</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>options.zh-cn.md</strong><dd><code>Update Python options documentation with specific line references </code><br><code>(Chinese)</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/drivers/options.zh-cn.md

<li>Updated Python code examples to use <code>gh-codeblock</code> with specific line <br>references.<br> <li> Replaced inline Python code with references to the test file.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1845/files#diff-9b23b749355db8154bfcc4a1d70f809b7d4e91a7b3044f9ac74a6c1a005873d6">+14/-28</a>&nbsp; </td>

</tr>                    
</table></details></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: examples/python/tests/drivers/test_options.py [hunk 1]
```diff
@@ -29,3 +31,69 @@ def test_page_load_strategy_none():
 
     driver.get("http://www.google.com")
     driver.quit()
+
```
Comment (pmartinez1) [inline examples/python/tests/drivers/test_options.py:34]: Anyone know how I can test these files locally? Local env points to upstream trunk.

Comment (harsha509) [reply examples/python/tests/drivers/test_options.py:34]: hi @pmartinez1 ,

Here are the instructions
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/python/README.md

Comment (harsha509) [reply examples/python/tests/drivers/test_options.py:34]: HI @pmartinez1 ,

Here are the instructions to run tests locally, https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/python/README.md

Comment (pmartinez1) [reply examples/python/tests/drivers/test_options.py:34]: Hi @harsha509 ,

I had followed those steps, but the links to the examples in my local server were pointing to the /trunk branch, so my newly added code-block examples were not showing up. I realized I needed to update the code-blocks to point to my local branch for those to be pulled in. Looks to be all there now. Ready for review.

Screen recording. 

https://github.com/user-attachments/assets/903c3952-d19b-4d1f-be4b-d27b7f7888a3

Comment (harsha509) [reply examples/python/tests/drivers/test_options.py:34]: Ah,

I usually commit code samples in one commit and then link to them in another commit. Occasionally, I carefully select line numbers from local files to include in the docs. This approach has consistently worked for me! :smile:

To answer your question, there's no other method available right now!

File: examples/python/tests/drivers/test_options.py [hunk 2]
```diff
@@ -29,3 +31,69 @@ def test_page_load_strategy_none():
...
+def test_capabilities():
+    options = webdriver.ChromeOptions()
+    options.browser_version = 'stable'
+    options.platform_name = 'any'
+    options.accept_insecure_certs = True
+    driver = webdriver.Chrome(options=options)
+
+    driver.get("http://www.google.com")
```
Comment (harsha509) [inline examples/python/tests/drivers/test_options.py]: can we use 'https://www.selenium.dev/'

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @pmartinez1,

Thank you for the PR!
Can you switch to using "https://www.selenium.dev/" in all the code samples?

Thanks,
Sri

File: examples/python/tests/drivers/test_options.py (hunk 2 above)
Comment (pmartinez1) [reply examples/python/tests/drivers/test_options.py]: I have updated them and will make sure to use selenium.dev moving forward.

**Review by harsha509 — APPROVED** [review]

**Comment by harsha509** [comment]
Thank you @pmartinez1!

Congratulations on your first contribution to Selenium Docs 🎉 🎉 🎉 !

**Pull request merged**